docs/asciidoc/routing.adoc

```diff
@@ -1207,25 +1209,99 @@ class Foo: Kooby({
 
 class App: Kooby({
 
-  use("/prefix", Foo())  // <1>
+  mount("/prefix", Foo())  // <1>
 
 })
 ----
 
 <1> Now all routes from `Foo` will be prefixed with `/prefix`. Output: `/prefix/foo` => `/prefix/foo`
 
-[TIP]
-====
-Composition is a great option for modularization. You can easily develop/test/deploy each 
-application independently and compose them all in another application.
+The mount operator only import routes. Services, callbacks, etc... are not imported. Main application
+is responsible for assembly all the resources and services required by imported applications.
+
+==== Install
+
+Alternative you can install a full application into a single one using the javadoc:Jooby[install, io.jooby.Jooby] operator:
+
+.Installing
+[source, java, role="primary"]
+----
+public class Foo extends Jooby {
+  {
+    get("/foo", ctx -> ...);
+  }
+}
+
+public class Bar extends Jooby {
+  {
+    get("/bar", ctx -> ...);
+  }
+}
+
+public class App extends Jooby {
+  {
+    install(Foo::new);                       // <1>
+
+    install(Bar::new);                       // <2>
+  }
+}
+----
+
+.Kotlin
+[source, kotlin,role="secondary"]
+----
+class Foo: Kooby({
+   
+  get("/foo") { ... }
+
+})
+
+class Bar: Kooby({
+  
+  get("/bar") { ... }
+
+})
+
+class App: Kooby({
+  install(::Foo)                         // <1>
+
+  install(::Bar)                         // <2>
+})
+----
+
+<1> Imports all routes, services, callbacks, etc... from `Foo`. Output: `/foo` => `/foo`
+
+<2> Imports all routes, services, callbacks, etc... from `Bar`. Output: `/bar` => `/bar`
+
+This operator let you to deploy for example `Foo` as standalone application or integrated into a main one `App`.
+Install operator shared the state of the main application, so lazy initialization of child application is *required*.
```

Comment:
Maybe it would be good to make it clear that this in practice means late **instantiation**.

> Install operator shares the state of the main application, so lazy initialization (and therefore _instantiation_) of any child application(s) is **mandatory**.